pkg/config/lpe_unix.go

```diff
@@ -0,0 +1,57 @@
+//go build unix
```

Comment:
This should look like this.
//go:build aix || darwin || dragonfly || freebsd || netbsd || openbsd || solaris || zos
// +build aix darwin dragonfly freebsd netbsd openbsd solaris zos

Reply:
Applied the change. Let's see if this will fix the issue.